blockchain/blockchain.go

```diff
@@ -922,13 +922,13 @@ func (bc *blockchain) startEmptyBlockchain() error {
 		PublicKey:  keypair.ZeroPublicKey,
 		RawAddress: addr.IotxAddress(),
 	}
+	if bc.sf == nil {
```

Comment:
We have unit test for startEmptyBlockchain() IIRC. Need to add test coverage for this.

Reply:
2chain e2e tests have covered this regression already